### PR TITLE
adding useR2021 call for tutorials

### PR DESCRIPTION
**Pull request by feddelegrand7**
Hi, 

I've added the call for tutorials in the highlight and the international sections. 

Thank you very much in advance. 
Best.

### CODE REVIEW

File: draft.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ image: https://rweekly.org/public/facebook.png
 ###  Highlight
 
 
++ [UseR2021: Call for Tutorials](https://user2021.r-project.org/participation/call-for-tutorials/)
```
Comment (BatoolMM) [inline draft.md]:
```suggestion

```
Thank you @feddelegrand7 for adding "the call for tutorials in UseR2021" in R Internationally section. It should have been added to this week's issue. Regarding Highlight section, the members of the R Weekly team vote for three to two posts on the R Weekly Slack channel every Saturday to decide which post goes in the highlight section. So, we don't fill in this section till the weekend (after the vote). Thank you again @feddelegrand7 .

**Review by BatoolMM — CHANGES_REQUESTED** [review]

File: draft.md (hunk 1 above)
Comment (feddelegrand7) [reply draft.md]: Hello @batoolMM thank you for your prompt reply. Ah yes I totally forgot the vote process, sorry for that. It's fine, even if it's not published in the highlights, having the call in the international section would be awesome. Thank you again :)

Comment (feddelegrand7) [reply draft.md]: I'm using my phone right now, I'll remove it from the highlights asap :)

Comment (BatoolMM) [reply draft.md]: no worries @feddelegrand7, I can commit the changes from my side.

Comment (feddelegrand7) [reply draft.md]: That would so nice. Thank you very much.

**Pull request merged**